engines/mysql.py

```diff
@@ -92,14 +93,18 @@ def table_exists(self, dbname, tablename):
                 self.existing_table_names.add((schema.lower(), table.lower()))
         return (dbname.lower(), tablename.lower()) in self.existing_table_names
 
-    def set_engine_encoding(self, encoding='ISO-8859-1'):
+    def set_engine_encoding(self):
         """Set MySQL database encoding to match data encoding
 
-        Defaults to latin1 and falls back to it if an unknown encoding is provided
-
+           Please update the encoding lookup table if the required encoding is not present.
         """
-        encoding_lookup = {'ISO-8859-1': 'latin1'}
-        db_encoding = encoding_lookup.get(encoding, 'latin1')
+
```

Comment:
Very nice work on this. One very minor here - all caps is used in Python to indicate constants - i.e. variables that never change value. So if we're going to assign a value to `ENCODING` in this way we should use `encoding` or something else lowercase instead.